manager/attestation_policy.go

```diff
@@ -72,40 +82,51 @@ func (ms *managerService) FetchAttestationPolicy(_ context.Context, computationI
 	case vmi.Config.EnableSEVSNP:
 		igvmMeasurementBinaryPath := fmt.Sprintf("%s/igvmmeasure", ms.attestationPolicyBinaryPath)
 
-		var stdoutBuffer bytes.Buffer
-		var stderrBuffer bytes.Buffer
-
 		stdout := bufio.NewWriter(&stdoutBuffer)
 		stderr := bufio.NewWriter(&stderrBuffer)
+		options := cmdconfig.IgvmMeasureOptions
 
-		igvmMeasurement, err := igvmmeasure.NewIgvmMeasurement(igvmMeasurementBinaryPath, stderr, stdout)
+		igvmMeasurement, err := cmdconfig.NewCmdConfig(igvmMeasurementBinaryPath, options, stderr, stdout)
 		if err != nil {
 			return nil, err
 		}
 
-		err = igvmMeasurement.Run(ms.qemuCfg.IGVMConfig.File)
+		outputByte, err := igvmMeasurement.Run(ms.qemuCfg.IGVMConfig.File)
 		if err != nil {
 			return nil, err
 		}
 
-		measurement = stdoutBuffer.Bytes()
+		outputString := string(outputByte)
+		lines := strings.Split(strings.TrimSpace(outputString), "\n")
+
+		if len(lines) == 1 {
+			outputString = strings.TrimSpace(outputString)
+			outputString = strings.ToLower(outputString)
+		} else {
+			return nil, fmt.Errorf("error: %s", outputString)
+		}
+
+		measurement, err = hex.DecodeString(outputString)
+		if err != nil {
+			return nil, err
+		}
 	}
 
 	if measurement != nil {
-		attestationPolicy.Policy.Measurement = measurement
+		attestationPolicy.Config.Policy.Measurement = measurement
 	}
 
 	if vmi.Config.SevConfig.EnableHostData {
 		hostData, err := base64.StdEncoding.DecodeString(vmi.Config.SevConfig.HostData)
 		if err != nil {
 			return nil, err
 		}
-		attestationPolicy.Policy.HostData = hostData
+		attestationPolicy.Config.Policy.HostData = hostData
 	}
 
-	attestationPolicy.Policy.MinimumLaunchTcb = vmi.LaunchTCB
+	attestationPolicy.Config.Policy.MinimumLaunchTcb = vmi.LaunchTCB
 
-	f, err = protojson.Marshal(&attestationPolicy)
+	f, err := json.MarshalIndent(attestationPolicy, "", " ")
```

Comment:
same here, all sections reading attestation policy will have to change from protojson